nilearn/datasets.py

```diff
@@ -217,10 +216,17 @@ def _get_dataset_dir(dataset_name, data_dir=None, lookup_defaults=False,
     # We build an array of successive paths by priority
     paths = []
 
+    # Search given environment variables
+    if env_vars is not None:
```

Comment:
If you set `env_vars` to `[]` by default, and this could be shortened to:
```python
        for env_var in env_vars:
            env_data = os.getenv(env_var, '')
            paths.extend(env_data.split(':'))
```